### ISSUE TITLE
solve_captchas.py doesn't work with sanitized module

### ISSUE THREAD

**Issue by eq2kazial** [Noctem/Monocle#161]
@Noctem  Getting the following error when trying to run solve_captchas.py:
```
authkey` not in config, and no default has been set.
Traceback (most recent call last):
  File "/Users/Scott/Monocle/monocle/sanitized.py", line 254, in __getattr__
    default = _defaults.pop(name)
KeyError: 'authkey'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "solve_captchas.py", line 168, in <module>
    loop.run_until_complete(main())
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/asyncio/base_events.py", line 466, in run_until_complete
    return future.result()
  File "solve_captchas.py", line 58, in main
    manager = AccountManager(address=get_address(), authkey=conf.authkey)
  File "/Users/Scott/Monocle/monocle/sanitized.py", line 260, in __getattr__
    raise AttributeError(err)
AttributeError: authkey not in config, and no default has been set.
```

**Comment by skateskate** [comment]
With the latest commit I've got this error when trying to run solve_captchas.py:
`aiopogo error on accountNameHere: InvalidPlatformRequest on GetPlayer.`

**Comment by NiliusJulius** [comment]
Same here
aiopogo error on accountNameHere: InvalidPlatformRequest on GetPlayer.

**Comment by baskettcase** [comment]
Yep I got the same error.

**Comment by Noctem** [comment]
@eq2kazial's issue was fixed in fe43dbbc34643b663ecd93d035dab5be59f23305

@skateskate, @NiliusJulius, and @baskettcase's was fixed in d3ecb6b5d89e61463bf38915059c28614e9abfe4.
